satpy/enhancements/__init__.py

```diff
@@ -110,6 +110,23 @@ def apply_enhancement(data, func, exclude=None, separate=False,
     return data
 
 
+# pointed to by generic.yaml
+def crefl_scaling(img, **kwargs):
```

Comment:
As discussed on slack I think we'll have to make another issue to combine this with the existing crude stretch so that it allows for multiple "control" points.